pkgs/by-name/op/optolith/package.nix

```diff
@@ -0,0 +1,111 @@
+{
+  stdenv,
+  requireFile,
+  autoPatchelfHook,
+  makeWrapper,
+  lib,
+
+  libGL,
+  udev,
+  alsa-lib,
+  at-spi2-atk,
+  cairo,
+  cups,
+  dbus,
+  expat,
+  gdk-pixbuf,
+  glib,
+  gtk3,
+  libdrm,
+  libgbm,
+  libxkbcommon,
+  nspr,
+  nss,
+  pango,
+  xorg,
+}:
+
+stdenv.mkDerivation rec {
+  name = "optolith";
+  version = "1.5.2";
+
+  src = requireFile rec {
+    name = "989002-Optolith_1.5.2.tar.gz";
+    hash = "sha256-CoGEsPxi8nZFJYdH51AgQVSrIFtlL6J6HYo8Lzp59Fg=";
+
+    message = ''
+      In order to install Optolith, you need to comply with the license agreement and download
+      the binaries, ${name}, from:
+
+      ${meta.downloadPage}
+
+      Once you have downloaded the file, please use the following command and re-run the
+      installation:
+
+      nix-prefetch-url file://\$PWD/${name}
+    '';
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    makeWrapper
+  ];
+
+  runtimeInputs = [
+    libGL.out
+    # leads to the following error if omitted:
+    # Zygote could not fork: process_type gpu-process numfds 3 child_pid -1
+    udev.out
+  ];
+
+  buildInputs = [
+    alsa-lib.out
+    at-spi2-atk.out
+    cairo.out
+    cups.lib
+    dbus.lib
+    expat.out
+    gdk-pixbuf.out
+    glib.out
+    gtk3.out
+    libdrm.out
+    libgbm.out
+    libxkbcommon.out
+    nspr.out
+    nss.out
+    pango.out
+    xorg.libX11.out
+    xorg.libXcomposite.out
+    xorg.libXdamage.out
+    xorg.libXext.out
+    xorg.libXfixes.out
+    xorg.libXrandr.out
+    xorg.libxcb.out
```

Comment:
`.out` is the default output, so AFAIK there's no need to mention it. I've not seen any other packages do this either.

Reply:
I copied this list from nix-alien. I had the same thought, but removing the `.out` pulled in a bunch more unneeded `.dev` outputs, so I thought why not. No strong opinion though, I can remove them :shrug:

Reply:
removed it. you're right, no other package does this, no reason to be weird.